airflow/www/blueprints.py

```diff
@@ -32,6 +35,22 @@ def index():
 
 @routes.route('/health')
 def health():
-    """ We can add an array of tests here to check the server's health """
-    content = Markup(markdown.markdown("The server is healthy!"))
-    return content
+    session = settings.Session()
+    DM = models.DagModel
+    payload = {}
+
+    payload['webserver'] = "The server is healthy!"
+
+    payload['database'] = "available"
+    try:
+        session.query(DM).count()
+    except Exception as _:
+        payload['database'] = "unavailable"
```

Comment:
Should this return something other than a 200 in this case? I'm not suer what is "typical" for a health check URL.

Reply:
Thanks @ashb .

I also noticed that people use status code to indicate the health check result in some applications. But the case here may be a bit different: Let's say webserver & database connection are both healthy, while scheduler is not working at all. What status code should we give?

So I would suggest to just give 200, and simply use the contents of the response to indicate the health of each component. Do you think it's ok?